keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -222,6 +243,7 @@ def get_random_transformation(
           image: 3D image tensor from inputs.
           label: optional 1D label tensor from inputs.
           bounding_box: optional 2D bounding boxes tensor from inputs.
+          segmentation_mask: optional 3D segmentation mask tensor from inputs.
 
         Returns:
           Any type of object, which will be forwarded to `augment_image`,
```

Comment:
Please update the return docstring for augment_segmentation_mask.

Reply:
Done